src/Illuminate/Database/Eloquent/ModelNotFoundException.php

```diff
@@ -24,6 +24,13 @@ class ModelNotFoundException extends RecordsNotFoundException
      */
     protected $ids;
 
+    /**
+     * The HTTP status code.
+     *
+     * @var int
+     */
+    protected int $status = 404;
```

Comment:
Wouldn't it make sense to extend the exception from [`\Symfony\Component\HttpKernel\Exception\HttpException`](https://github.com/symfony/symfony/blob/7.3/src/Symfony/Component/HttpKernel/Exception/HttpException.php) then?